commit 8db0137dc390790175e6f2a9073eda822ce4179c
Author: John Spray <jspray@redhat.com>
Date:   Mon Aug 8 17:08:42 2016 +0100

    mgrc: less vebose logging of messages
    
    ...no need to tell us about all the things you're *not* handling.
    
    Signed-off-by: John Spray <john.spray@redhat.com>

diff --git a/src/mgr/MgrClient.cc b/src/mgr/MgrClient.cc
index 5dfb8de7706..d0baeb4d374 100644
--- a/src/mgr/MgrClient.cc
+++ b/src/mgr/MgrClient.cc
@@ -59,7 +59,6 @@ bool MgrClient::ms_dispatch(Message *m)
 {
   Mutex::Locker l(lock);
 
-  ldout(cct, 20) << *m << dendl;
   switch(m->get_type()) {
   case MSG_MGR_MAP:
     return handle_mgr_map(static_cast<MMgrMap*>(m));
@@ -73,7 +72,7 @@ bool MgrClient::ms_dispatch(Message *m)
       return false;
     }
   default:
-    ldout(cct, 10) << "Not handling " << *m << dendl; 
+    ldout(cct, 30) << "Not handling " << *m << dendl; 
     return false;
   }
 }
@@ -82,6 +81,8 @@ bool MgrClient::handle_mgr_map(MMgrMap *m)
 {
   assert(lock.is_locked_by_me());
 
+  ldout(cct, 20) << *m << dendl;
+
   map = m->get_map();
   ldout(cct, 4) << "Got map version " << map.epoch << dendl;
   m->put();
@@ -244,6 +245,8 @@ bool MgrClient::handle_mgr_configure(MMgrConfigure *m)
 {
   assert(lock.is_locked_by_me());
 
+  ldout(cct, 20) << *m << dendl;
+
   if (session == nullptr) {
     lderr(cct) << "dropping unexpected configure message" << dendl;
     m->put();
@@ -313,6 +316,8 @@ bool MgrClient::handle_command_reply(MCommandReply *m)
 {
   assert(lock.is_locked_by_me());
 
+  ldout(cct, 20) << *m << dendl;
+
   const auto tid = m->get_tid();
   const auto op = command_table.get_command(tid);
   if (op == nullptr) {
